samples/testing/dump_operation/test_dump_operation.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
To match the project renaming suggestion, I'd rename this file to "test_operations.py", since it showcases both approaches